packages/headless/src/features/commerce/facets/facet-set/facet-set-reducers.ts

```diff
@@ -55,6 +59,27 @@ function restoreRegularFacets(
   }
 }
 
+function restoreLocationFacets(
```

Comment:
You could make a generic function for both regular and location facets to avoid dulication here (kind of like we have for range facets).